src/components/learner-credit-management/cards/NewAssignmentModalButton.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FullscreenModal,
+  ActionRow,
+  Button,
+  useToggle,
+  Hyperlink,
+} from '@edx/paragon';
+import AssignmentModalContent from './AssignmentModalContent';
+
+const NewAssignmentModalButton = ({ course, children }) => {
+  const [isOpen, open, close] = useToggle(false);
+
+  return (
+    <>
+      <Button onClick={open}>{children}</Button>
+      <FullscreenModal
+        className="bg-light-200 text-left"
+        title="Assign this course"
+        isOpen={isOpen}
+        onClose={close}
+        footerNode={(
+          <ActionRow>
+            <Button variant="tertiary" as={Hyperlink} destination="https://edx.org" target="_blank">
+              Help Center: Course Assignments
+            </Button>
+            <ActionRow.Spacer />
+            <Button variant="tertiary" onClick={close}>Cancel</Button>
+            <Button>Assign</Button>
```

Comment:
**[question] (non-blocking)**: Is there a ticket to hook up the functionality to this button? if so, can we leave a TO-DO comment here?

Reply:
Ah, I opted not to include a TODO comment here with [the ticket](https://2u-internal.atlassian.net/browse/ENT-7826) as I picked it up as my current, in-progress ticket after this PR. Good callout, though!